lib/content-services/src/lib/search/components/search-text/search-text.component.scss

```diff
@@ -3,25 +3,38 @@
 .adf-search-text {
     padding: 0;
 
-    #{$mat-form-field-wrapper} {
-        margin-top: -5px;
-        padding: 0;
+    .adf-search-text-form-field {
+        width: 100%;
+        height: 58.5px;
+        margin-top: 5px;
 
-        #{$mat-form-field-infix} {
-            padding-bottom: 6px;
+        &-clear-button {
+            min-width: unset;
+            border-radius: 50%;
         }
-    }
 
-    #{$mat-notched-outline} {
-        #{$mat-notched-outline-trailing},
-        #{$mat-notched-outline-notch} {
-            border-top: 0 solid;
-            border-left: 0 solid;
-            border-right: 0 solid;
-            border-bottom: 1px solid var(--adf-theme-foreground-secondary-text-color);
-
-            #{$mat-floating-label} {
-                left: 0;
+        #{$mat-form-field-wrapper},
```

Comment:
We  want to switch away from mat selectors. Since this is a component specific css file, is it not possible to use something like classes?

Reply:
it's inner element of angular material, so we can't add any custom class to it. It's mostly reformatting, as it was already added here